package.json

```diff
@@ -20,7 +20,9 @@
   ],
   "scripts": {
     "build": "rm -rf lib/** && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json && npm run api-extractor && npm run generate-docs && npm run generate-notices && vite build",
-    "lint": "eslint .",
+    "prepare": "husky install",
```

Comment:
Does this mean everyone installing the library needs to run npm prepare in order for the commit hooks to work?

Didn't quite follow what you meant by you need to run prepare.

Reply:
In order to use commit hook, it’s required to run ‘husky install’. Just install the library is not enough.

Reply:
I don't understand. If the commit hook is working, then the entire repo should be formatted, and we should be seeing much more being changed in this CR. 

Additionally, the build is failing due to the formatting being off, which this commit cook is supposed to fix before it's merged.

Reply:
See here: https://github.com/yext/analytics/pull/86

Reply:
So is this just to add the hook? And another is to just reformat the world?

Reply:
yes